packages/bot-web-ui/src/components/quick-strategy.jsx

```diff
@@ -62,77 +64,141 @@ const QuickStrategyForm = ({
-                                            onHideDropdownList={() => {
-                                                onHideDropdownList('symbol', values[field.name], setFieldValue);
-                                            }}
-                                            onItemSelection={({ value }) => {
-                                                onChangeDropdownItem('symbol', value, setFieldValue);
-                                            }}
-                                            onScrollStop={() => onScrollStopDropdownList('symbol')}
-                                            leading_icon={
-                                                selected_symbol.value && (
-                                                    <Icon icon={`IcUnderlying${selected_symbol.value}`} size={24} />
-                                                )
-                                            }
-                                        />
+                                        <>
+                                            {!is_mobile ? (
+                                                <Autocomplete
+                                                    {...field}
+                                                    autoComplete='off'
+                                                    className='quick-strategy__dropdown quick-strategy__leading'
+                                                    type='text'
+                                                    label={localize('Asset')}
+                                                    list_items={symbol_dropdown}
+                                                    onHideDropdownList={() => {
+                                                        onHideDropdownList('symbol', values[field.name], setFieldValue);
+                                                    }}
+                                                    onItemSelection={({ value }) => {
+                                                        onChangeDropdownItem('symbol', value, setFieldValue);
+                                                    }}
+                                                    onScrollStop={() => onScrollStopDropdownList('symbol')}
+                                                    leading_icon={
+                                                        selected_symbol.value && (
+                                                            <Icon
+                                                                icon={`IcUnderlying${selected_symbol.value}`}
+                                                                size={24}
+                                                            />
+                                                        )
+                                                    }
+                                                />
+                                            ) : (
+                                                <Dropdown
```

Comment:
I think u meant to use `SelectNative` here? `Dropdown` is only meant to be used on Desktop.

Reply:
I think you didn't see my last commit, in that one I replaced all `<dropdown>s` with `SelectNative`.

Reply:
I see, because during my review comment I can still see that `<Dropdown>` was used as shown above, but if it's updated already it's all good. 👍🏻